aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,31 +170,54 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are five types of capitalization. 
+1. Sentence case, where the first letter is capitalized and subsequent words begin with lowercase letters, with some exceptions, such as proper nouns.
+   (example: Storage volume) 
+2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
+   (example: "The Benefits of Cloud Storage Explained")
+3. PascalCase, where there are capital letters inside the word, almost always for branding purposes.
+    (example: GreenLake)
+4. All lower case, where every single word is lower case 
+    (example: "lower case exudes an approachable and casual vibe")
+5. All caps, where every letter is capitalized
+    (example: ALL LETTERS ARE CAPITALIZED)
+
+At HPE, almost all text should be written in sentence case. Never begin a sentence or phrase with a lower case letter because it is too casual for our brand. Do not write in all caps because it is equated with yelling and increases stress.
+
+Sentence case should be applied on:
+- Attribute labels
+- Column headings
+- Legends
+- Tabs
+- Menus
+- Section and subsection headings
 - Paragraphs of text
 - Tips
 - Notifications
 - Help
+- Buttons
```

Comment:
I believe previously we've been told that Button labels like "Sign Up" etc. should use title case -- has that guidance been modified? If so, we'll need to modify our site examples and a dev ticket should be created.

Reply:
Button case should match the case of the button target.  And, because page titles and dialog titles should be sentence case, the buttons should also be sentence case.

Reply:
I was able to find the [initial ticket and discussion](https://github.com/grommet/hpe-design-system/issues/937#issuecomment-651214534) where we updated certain button labels like "Sign Up" to use title case.

Based on this screen grab from Brand Central's Writing Style Guide, it seems that there are some use cases where it is appropriate to title case a button label.

<img width="903" alt="Screen Shot 2022-08-26 at 1 06 15 PM" src="https://user-images.githubusercontent.com/12522275/186983309-eba82f7a-e614-45d2-a6e9-76543b2cadcf.png">


I agree with you Edgar that button case should match the case of the button target. Do you think it would be helpful to highlight some of the cases that aren't sentence case so that there's more clarity?

Reply:
If there is a general rule with limited specific exceptions, we should list the exceptions.

Reply:
@dahliaferrer we should highlight these limited specific exceptions.